spec/features/climate_home_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+RSpec.describe "Climate Home", type: :feature do
```

Comment:
I *think* you can drop the `type: :feature` here because rspec infers that it's a feature spec because it's in the feature folder.

We have `infer_spec_type_from_file_location!` set in the setup file.